esx_service/volume_kv.py

```diff
@@ -82,7 +84,7 @@
 ACCESS_TYPES = [ACCESS_READWRITE, ACCESS_READONLY]
 
 # Filesystem type
-# This option is handled in the volume-plugin at the docker host, and tracked in volume metadata. 
+# This option is handled in the volume-plugin at the docker host, and tracked in volume metadata.
```

Comment:
Hi @msterin,

If you say so. I was annoyed I couldn't find a single editor to save that file that didn't also nuke that trailing character. That's what actually bugged me.